src/instantx.h

```diff
@@ -235,6 +235,7 @@ class CTxLockVote
     COutPoint outpoint;
     // TODO remove this member when the legacy masternode code is removed after DIP3 deployment
     COutPoint outpointMasternode;
+    uint256 outpointConfirmedBlock;
```

Comment:
The name is misleading imo, this is neither an outpoint nor a block.

Reply:
But it's the hash of the block that confirmed the IX input (mined block + nInstantSendConfirmationsRequired - 1). Do you have a better suggestion for the name?

Reply:
I used the term "modifier" in LLMQs when feeding the quorum calculation argument. Maybe `quorumModifierHash` is a good name here as well?